Test/model_integration_tests/model_integration_test.ts

```
@@ -161,5 +162,51 @@ describe('model methods', () => {
       assertEquals(Shrike.rows, [{ name: 'Shrike' }]);
       assertEquals(Consul.rows, [{ name: 'The Consul' }]);
     });
+
+    it('it should throw an error because of a lack of a space around equal sign', async () => {
+      await assertRejects(async () => {
```

a whole transaction should throw an error if a single query in the transaction is malformed